src/explore-education-statistics-frontend/src/modules/permalink/PermalinkPage.tsx

```diff
@@ -59,100 +66,14 @@ class PermalinkPage extends Component<Props> {
-              <th
-                className="govuk-table__header govuk-table__cell--numeric"
-                scope="col"
-              >
-                2015/16
-              </th>
-              <th
-                className="govuk-table__header govuk-table__cell--numeric"
-                scope="col"
-              >
-                2016/17
-              </th>
-            </tr>
-          </thead>
-          <tbody className="govuk-table__body">
-            <tr className="govuk-table__row ">
-              <th className="govuk-table__header">Unauthorised absence rate</th>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                1.1
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                1.1
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                1.1
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                1.1
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                1.3
-              </td>
-            </tr>
-            <tr className="govuk-table__row ">
-              <th className="govuk-table__header">Overall absence rate</th>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                5.3
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                4.5
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                4.6
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                4.6
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                4.7
-              </td>
-            </tr>
-            <tr className="govuk-table__row ">
-              <th className="govuk-table__header">Authorised absence rate</th>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                4.2
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                3.5
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                3.5
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                3.4
-              </td>
-              <td className="govuk-table__cell govuk-table__cell--numeric">
-                3.4
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <TimePeriodDataTable
+          fullTable={fullTable}
+          tableHeadersConfig={getDefaultTableHeaderConfig(
+            fullTable.subjectMeta,
+          )}
+        />
+        <DownloadCsvButton publicationSlug="testy-boi" fullTable={fullTable} />
```

Comment:
testy-boi?

Reply:
Ahh, copied from a test-page I created to test with dummy data!

Nice catch 🎣